docs/tools/vdb_table/data/databricks.json

```diff
@@ -0,0 +1,188 @@
+{
+    "name": "Databricks Vector Search",
+    "links": {
+      "docs": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "github": "",
+      "website": "https://www.databricks.com/product/machine-learning/vector-search",
+      "vendor_discussion": "",
+      "poc_github": "",
+      "slug": "databricks"
+    },
+    "oss": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "license": {
+      "value": "Proprietary",
+      "source_url": "",
```

Comment:
would be good to have a link to the proprietary license here

Reply:
Done